app/views/messages/_messages_table.html.erb

```diff
@@ -0,0 +1,13 @@
+<table class="table table-sm align-middle">
+  <thead>
+    <tr>
+      <th><%= t ".from" %></th>
+      <th><%= t ".subject" %></th>
+      <th><%= t ".date" %></th>
+      <th class="d-flex justify-content-end"><%= t ".actions" %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <%= render :partial => inner_partial, :collection => messages %>
```

Comment:
There is a level of indirection here with isn't used, since "inner_partial" is always equal to "message_summary". I guess what might have happened is that you started looking at the two different contents for this table - inbox and outbox being slightly different - but you haven't actually refactored the outbox, perhaps because the headings are different too.

So I think either more refactoring is required (so that the inner_partial is useful) or this level of indirection can be removed.

Reply:
see c0cbf1c28da68739bc854eb8249f5298ce16e8a5